### PR TITLE
fix linter issues to be more in line with java rules

### PR DESCRIPTION
**Pull request by feddevanderlist**
Fixing some linter issues based on the java-sonarcube-default rule set.

### CODE REVIEW

**Comment by feddevanderlist** [comment]
@jarlah I was wondering if for the tests we could put the comments in the code in java doc. That way code gets a little cleaner and more readable. :)

**Comment by jarlah** [comment]
You can move comments thats ok. But can you add automated code linting in workflow? If it possible to come to a state when where it breaks it dont show a list of 100 problems ?

**Comment by feddevanderlist** [comment]
> You can move comments thats ok. But can you add automated code linting in workflow? If it possible to come to a state when where it breaks it dont show a list of 100 problems ?

I don't know about a way in github to do that. But I'll take a look to find out if it's possible :)

**Comment by feddevanderlist** [comment]
https://www.freecodecamp.org/news/github-super-linter/ this might be an option :)

**Comment by jarlah** [comment]
maybe you can test it out on your own fork. I see it requires a bit of a setup with secrets and stuff

**Comment by feddevanderlist** [comment]
I'll take a look I've never used github actions before. I'm a gitlab person.

**Comment by jarlah** [comment]
Chatgpt or perplexity ftw 🙈

**Comment by jarlah** [comment]
you can do that in different PR if you want. Just mark it ready when you have fixed the linting stuff you seem fitting

**Comment by jarlah** [comment]
might also be overly overkill with github action for linting java code when there is literally almost no code

**Review by jarlah — APPROVED** [review]

File: src/test/java/org/testcontainers/containers/CephContainerTest.java [hunk 1]
```diff
@@ -43,9 +43,8 @@ class CephContainerTest {
     @Test
     void testBasicUsage() throws Exception {
         try (
-                // minioContainer {
+                // minioContainer
```
Comment (jarlah) [inline src/test/java/org/testcontainers/containers/CephContainerTest.java:46]: <3

File: src/test/java/org/testcontainers/containers/CephContainerTest.java [hunk 2]
```diff
@@ -192,7 +198,7 @@ void testOverrideWaitingFor() {
             assertThat(container.getWaitStrategy()).isInstanceOf(HostPortWaitStrategy.class);
         }
     }
-    // configuringClient {
+    // configuringClient
```
Comment (jarlah) [inline src/test/java/org/testcontainers/containers/CephContainerTest.java:201]: aaaactually you can remove these comments cant you .. unless they give some new meaning ?

**Review by jarlah — APPROVED** [review]
Looks good :) 

i would recommend removing the silly one word "comment blocks" though

File: src/test/java/org/testcontainers/containers/CephContainerTest.java [hunk 3]
```diff
@@ -76,41 +74,41 @@ public void testBasicUsage() throws Exception {
...
 
             ResponseInputStream<GetObjectResponse>
-                responseInputStream = s3client.getObject(request -> request.bucket("demo").key(key));
+                    responseInputStream = s3client.getObject(request -> request.bucket("demo").key(key));
             assertThat(responseInputStream).hasContent("This is a file");
 
-            ListObjectsV2Response objets = s3client.listObjectsV2((builder) -> builder.bucket("demo"));
-            assertThat(objets.contents().size()).isEqualTo(1);
+            ListObjectsV2Response objets = s3client.listObjectsV2(builder -> builder.bucket("demo"));
+            assertThat(objets.contents()).hasSize(1);
```
Comment (jarlah) [inline src/test/java/org/testcontainers/containers/CephContainerTest.java:81]: <3

File: src/test/java/org/testcontainers/containers/CephContainerTest.java [hunk 4]
```diff
@@ -76,41 +74,41 @@ public void testBasicUsage() throws Exception {
...
+    void testOverrides() throws Exception {
         try (
-            // cephOverrides {
-            CephContainer container = new CephContainer("quay.io/ceph/demo:latest")
-                .withCephAccessKey("testuser123")
-                .withCephSecretKey("testpassword123")
-                .withCephBucket("testbucket123")
-                .withSslDisabled()
-            // }
+                // cephOverrides
```
Comment (jarlah) [inline src/test/java/org/testcontainers/containers/CephContainerTest.java:98]: remove, lies in test method name

**Pull request merged**